<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>番茄钟</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="focusTab">
       <attribute name="title">
        <string>专注</string>
       </attribute>
       <layout class="QVBoxLayout" name="focusLayout">
        <item>
         <widget class="QGroupBox" name="taskGroupBox">
          <property name="title">
           <string>任务</string>
          </property>
          <layout class="QHBoxLayout" name="taskSelectLayout">
           <item>
            <widget class="QLabel" name="taskLabel">
             <property name="text">
              <string>选择任务:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="taskComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="timeGroupBox">
          <property name="title">
           <string>时间设置</string>
          </property>
          <layout class="QGridLayout" name="timeSettingsLayout">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="workTimeLabel">
             <property name="text">
              <string>工作时间（分钟）:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="workMinSpinBox">
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>60</number>
             </property>
             <property name="value">
              <number>25</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="breakTimeLabel">
             <property name="text">
              <string>休息时间（分钟）:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="breakMinSpinBox">
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>60</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="cyclesLabel">
             <property name="text">
              <string>番茄个数:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="cyclesSpinBox">
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="progressGroupBox">
          <property name="title">
           <string>进度</string>
          </property>
          <layout class="QVBoxLayout" name="progressLayout">
           <item>
            <widget class="QLabel" name="statusLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="controlGroupBox">
          <property name="title">
           <string>控制</string>
          </property>
          <layout class="QHBoxLayout" name="controlLayout">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="text">
              <string>开始</string>
             </property>
             <property name="icon">
              <iconset theme="media-playback-start"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pauseButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>暂停</string>
             </property>
             <property name="icon">
              <iconset theme="media-playback-pause"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetButton">
             <property name="text">
              <string>重置</string>
             </property>
             <property name="icon">
              <iconset theme="view-refresh"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="projectTab">
       <attribute name="title">
        <string>任务管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="projectLayout">
        <item>
         <layout class="QHBoxLayout" name="taskButtonLayout">
          <item>
           <widget class="QPushButton" name="addTaskButton">
            <property name="text">
             <string>添加任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editTaskButton">
            <property name="text">
             <string>编辑任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTaskButton">
            <property name="text">
             <string>删除任务</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="taskList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <widget class="QGroupBox" name="soundGroupBox">
          <property name="title">
           <string>音效设置</string>
          </property>
          <layout class="QVBoxLayout" name="soundLayout">
           <item>
            <layout class="QHBoxLayout" name="workSoundLayout">
             <item>
              <widget class="QLabel" name="workSoundLabel">
               <property name="text">
                <string>专注音效:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="workSoundEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseWorkSoundButton">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testWorkSoundButton">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="breakSoundLayout">
             <item>
              <widget class="QLabel" name="breakSoundLabel">
               <property name="text">
                <string>休息音效:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="breakSoundEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseBreakSoundButton">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testBreakSoundButton">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="completeSoundLayout">
             <item>
              <widget class="QLabel" name="completeSoundLabel">
               <property name="text">
                <string>完成音效:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="completeSoundEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseCompleteSoundButton">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testCompleteSoundButton">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="soundEnableLayout">
             <item>
              <widget class="QCheckBox" name="enableWorkSoundCheck">
               <property name="text">
                <string>启用专注音效</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableBreakSoundCheck">
               <property name="text">
                <string>启用休息音效</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableCompleteSoundCheck">
               <property name="text">
                <string>启用完成音效</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="settingsVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statsTab">
       <attribute name="title">
        <string>时间统计</string>
       </attribute>
       <layout class="QVBoxLayout" name="statsLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QWidget" name="totalTimeWidget" native="true">
          <layout class="QHBoxLayout" name="totalTimeLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_left">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="totalTimeTitle">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>累计专注时长:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalTimeLabel">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_right">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="taskListContainer" native="true">
          <layout class="QVBoxLayout" name="taskListContainerLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QScrollArea" name="taskScrollArea">
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>351</width>
                <height>288</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="taskItemsLayout">
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
